test/tests/composer.js

```diff
@@ -1225,23 +1225,23 @@ module.exports = Nodal => {
             children__is_favorite: true,
             children__license: null,
             pets__name: 'Oliver',
-            pets__alive: true,
```

Comment:
I would write a new test case here instead of adapting an old one

Reply:
This is actually an issue with the existing tests since the column `active` doesn't exist (it's `is_active`). I had to update them because my new changes would break the tests otherwise. This column doesn't seem to affect the result of the test anyway since they still pass like this.

Reply:
Ah gotcha. Makes sense then